src/docgen/render/markdown.ts

```diff
@@ -177,7 +193,7 @@ export class Markdown {
           `${heading} <span data-heading-title="${this.header}" data-heading-id="${anchor}"></span>`,
         );
       } else {
-        content.push(`${heading} <a name="${this.id}"></a>`);
+        content.push(`${heading} <a name="${this.id}" id="${anchor}"></a>`);
```

Comment:
I am not sure but I think that maybe we want the anchor just before the heading so when a link is clicked the viewport will scroll so the heading is visible. I never figured out the right way to do this. Maybe @gabewomble has some insight

Reply:
It seems like the headings are visible when I click on anchored links on the gist or on Construct Hub, is this maybe a browser specific thing? Maybe create an issue about it.

If I understand your issue correctly, it might be better fixed with some CSS perhaps https://stackoverflow.com/questions/11501025/div-anchors-scrolling-too-far